### PR TITLE
Add storm node.props.set (SYN-2481)

### PR DESCRIPTION
**Pull request by rakuy0**

### CODE REVIEW

File: synapse/lib/stormtypes.py [hunk 1]
```diff
@@ -3964,6 +3972,9 @@ async def iter(self):
         for item in items:
             yield item
 
+    async def set(self, prop, valu):
+        return await self.valu.set(prop, valu)
```
Comment (invisig0th) [inline synapse/lib/stormtypes.py:3976]: You'll need to add perms checks against the layer here :) It's normally handled by the storm AST but methods are on their own :)

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: synapse/lib/stormtypes.py (hunk 1 above)
Comment (invisig0th) [reply synapse/lib/stormtypes.py:3976]: ( check out how node.data.set() does it, it'll be similar )

Comment (invisig0th) [reply synapse/lib/stormtypes.py:3976]: Oh, and dont forget to add a test that confirms that it can explode with AuthDeny :D

Comment (rakuy0) [reply synapse/lib/stormtypes.py:3981]: :+1:

**Review by invisig0th — APPROVED** [review]

**Pull request merged**